net/yggdrasil/files/ygguci

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env lua
```

Comment:
This comment is out of scope.
But could we please convert <spaces> into <tabs> in this file?
OpenWrt always uses <tabs>.
This reduces the file size.